src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -174,10 +199,27 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         const fileVersionId = getProp(match, 'params.fileVersionId', currentFileVersionId);
 
         history.push(
-            activeAnnotationId ? `/activity/annotations/${fileVersionId}/${activeAnnotationId}` : '/activity/',
+            activeAnnotationId
+                ? `/activity/annotations/${fileVersionId}/${activeAnnotationId}`
+                : `/activity/annotations/${fileVersionId}`,
         );
     };
 
+    handleVersionChange = () => {
+        const { file, history, match, onVersionChange } = this.props;
+        const { feedItems = [] } = this.state;
```

Comment:
Should `feedItems` value be defaulted in the constructor?

Reply:
Seems like there's some logic in downstream components that cares if feedItems is `undefined` vs empty so I'll leave it as is for now